src/velbustcp/lib/connection/tcp/network.py

```diff
@@ -59,39 +65,92 @@ def send(self, data: bytearray) -> None:
                 except Exception:
                     self.__logger.exception("Could not send data to client %s", client.address())
 
-    def __accept_sockets(self) -> None:
-        """Accepts clients from socket, if the tcp server is closed it will also close socket
-        """
+    def __get_bound_socket(self) -> socket.socket:
```

Comment:
```suggestion
    def __get_bound_socket(self) -> Optional[socket.socket]:
```

If we didn't start the server yet, this call will return None